looker_blueprints/download_look_as_file.py

```diff
@@ -0,0 +1,75 @@
+import argparse
+import sys
+import os
+import shipyard_utils as shipyard
+try:
+    import helpers
+except BaseException:
+    from . import helpers
+
+
+EXIT_CODE_LOOK_ERROR = 204
+
+def get_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--base-url', dest='base_url', required=True)
+    parser.add_argument('--client-id', dest='client_id', required=True)
+    parser.add_argument('--client-secret', dest='client_secret', required=True)
+    parser.add_argument('--look-id', dest='look_id', required=False)
```

Comment:
Look-id should be required unless I am missing a functionality that would make it optional